src/fontra/views/editor/edit-behavior.js

```diff
@@ -22,6 +22,8 @@ export class EditBehaviorFactory {
       point: pointSelection,
       component: componentSelection,
       anchor: anchorSelection,
+      guidelineGlyph: guidelineGlyphSelection,
```

Comment:
This name is awkward, there is no such thing as a "guideline glyph". I would call it "guideline", and for the globas/font guidelines we can later use "fontGuideline".